Doc/whatsnew/3.7.rst

```diff
@@ -2401,6 +2401,10 @@ Changes in the Python API
   instead of a :class:`bytes` instance.
   (Contributed by Victor Stinner in :issue:`21071`.)
 
+* :mod:`argparse` subparsers can now be made mandatory by passing ``required=True``
+  to :meth:`ArgumentParser.add_subparsers() <argparse.ArgumentParser.add_subparsers>`.
```

Comment:
Small thing I didn’t notice before merge: in func/meth/etc roles, we don’t include the parentheses.  Sphinx has a setting to add them or not automatically to all roles.

(I personnally don’t like to have them, especially given Python’s clear difference of referencing a function vs. calling it, but many programmers are used to writing things like `the whatever() function` probably to make it less ambiguous in plain text contexts like email, so Python’s doc is set up to have them :slightly_smiling_face:)

I didn’t pay attention to the fact that the section here (`Changes in the Python API`) was part of `Porting to Python 3.7` (we really need docs previews for PRs!), which made sense in the original PR (changed argparse behaviour) but less so now.  I think it would have been better in `Improved Modules` → `argparse` https://docs.python.org/3.9/whatsnew/3.7.html#argparse